addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -87,6 +90,7 @@ def _mark_conflicting_work_entries(self, start, stop):
         """
         Set `state` to `conflict` for overlapping work entries
         between two dates.
+        If `ids` is specified then check conflicts with the corresponding work entries.
```

Comment:
```suggestion
```

Reply:
I was just pushing the last change, still a comment on the docstring but clearer